packages/quantic/force-app/main/default/lwc/quanticRadioButtonsGroup/__tests__/quanticRadioButtonsGroup.test.js

```diff
@@ -0,0 +1,289 @@
+import QuanticRadioButtonsGroup from 'c/quanticRadioButtonsGroup';
+// @ts-ignore
+import {createElement} from 'lwc';
+
+const selectors = {
+  legend: 'legend.radio-buttons-group__legend',
+  radioInputs: 'input[type="radio"]',
+  radioLabels: 'label.radio-buttons-group__label',
+  radioIcons: 'lightning-icon',
+  radioContainers: '.slds-button.slds-radio_button.radio-button',
+  radioTooltips: 'c-quantic-tooltip',
+};
+
+const defaultOptionsList = [
```

Comment:
This will allow you to well define the typings and avoid adding ts-ignors in the code.
```suggestion
/**
 * @type {Array<{value: string, tooltip?: string, label?: string }>}
 */
const defaultOptionsList = [
```